main.tf

```diff
@@ -19,8 +19,9 @@ locals {
 }
 
 resource "google_pubsub_topic" "topic" {
-  name    = var.topic
   project = var.project_id
+  name    = var.topic
+  labels  = var.topic_labels
```

Comment:
Has this been tested for backward compatibility? We need to be sure that adding labels to an existing resource isn't a destructive operation.

Reply:
I've tested backward compatibility. Adding labels case update in-place, without any destruction of previously-created resources: 
![image](https://user-images.githubusercontent.com/15910313/63869952-af64ca00-c9c1-11e9-8983-e36cdbc8e77c.png)